packages/slice-machine/lib/models/server/Client/upload.ts

```diff
@@ -2,19 +2,21 @@ import fs from "fs";
 import FormData from "form-data";
 import mime from "mime";
 
-function upload({
+export type UploadParams = {
+  url: string;
+  fields: Record<string, string>;
+  key: string;
+  filename: string;
+  pathToFile: string;
+};
+
+export function upload({
   url,
   fields,
   key,
   filename,
   pathToFile,
-}: {
-  url: string;
-  fields: { [key: string]: string };
-  key: string;
-  filename: string;
-  pathToFile: string;
-}): Promise<number | undefined> {
+}: UploadParams): Promise<number | undefined> {
```

Comment:
I didn't modify the code of this one actually, but I can imagine that if we return a number it's because we have a result and an undefined could mean we had a 500.